packages/web3-eth/src/utils/prepare_transaction_for_signing.ts

```diff
@@ -125,19 +125,19 @@ export const prepareTransactionForSigning = async (
 	web3Context: Web3Context<EthExecutionAPI>,
 	privateKey?: HexString | Uint8Array,
 	fillGasPrice = false,
+	fillGas = true,
```

Comment:
```suggestion
	fillGasLimit = true,
```

gas and gaslimit are same , it might be confusing with gasPrice, so suggesting to use gaslimit